packages/common/src/ui/components/buttons/IconButton.stories.tsx

```diff
@@ -2,21 +2,24 @@ import React from 'react';
 import { Box } from '@mui/material';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
 import { IconButton } from './IconButton';
-import { Book } from '../../icons';
+import { Book, SvgIconProps } from '../../icons';
 
-const Template: ComponentStory<typeof IconButton> = () => (
+const Template: ComponentStory<React.FC<SvgIconProps>> = args => (
   <Box>
     <IconButton
-      icon={<Book />}
-      titleKey="button.docs"
+      icon={<Book {...args} />}
+      labelKey="button.docs"
```

Comment:
It seemed like half the buttons were using `labelKey` and half `titleKey` (Although it might have actually been more in favour of the one I didn't choose) so I tried to consolidate on a single one

Reply:
oh oops.